### PR TITLE
Fix/prep start

### PR DESCRIPTION
**Pull request by ptbarnum4**
- Minor updates to MacCard element
- Fix page jitter on MacCard resize/transition
- Change "Prep start date" → "Next Info Session Date"
  - **TODO:** Update greenlight webhook for info session dates to remove the upcoming date/time 1 hour prior to start
- Add an optional `infoMessage` property to course info, designed only to be used for prep. 
   > "Attending info session and orientation is required prior to starting prep"

### CODE REVIEW

File: pages/programs/workforce.tsx [hunk 1]
```diff
@@ -2,6 +2,7 @@ import { useEffect, useState } from 'react';
 import { GetStaticProps, NextPage } from 'next';
 import Link from 'next/link';
 import styled, { useTheme } from 'styled-components';
+import moment from 'moment';
```
Comment (harveysanders) [inline pages/programs/workforce.tsx:5]: This is just a suggestion:
`moment` is relatively huge compared to some alternatives. If you only need a few helpers from it, you might consider another lib, like `dayjs` or `date-fns`.
https://vhudyma-blog.eu/the-best-momentjs-alternatives/

File: pages/programs/workforce.tsx [hunk 2]
```diff
@@ -97,53 +105,60 @@ const AdultPrograms: NextPage<AdultProgramsProps> = ({
...
-                        className='anchor right-arr-left'
-                        aria-label='Contact to learn about employer partnerships'
-                        title='Contact to learn about employer partnerships'
+                      <MacContent
+                        body={quote.body}
+                        imageUrl={quote.imageUrl}
+                        name={quote.name}
+                        role={quote.role}
+                        logoSrc={theme.isLightMode ? quote.logoSrcLight : quote.logoSrcDark}
+                        logoHref='https://mumms.com/'
```
Comment (harveysanders) [inline pages/programs/workforce.tsx:126]: Should the logo URL be in the `quote` as well?

File: src/hooks/useInfoSession.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+function useInfoSession(nextOnly?: boolean) {
+  const [sessionDates, setSessionDates] = useState<ISessionDates[]>([]);
+
+  useEffect(() => {
+    axios.get('/api/infoSession/dates').then(({ data }) => setSessionDates(data));
+  }, []);
+
+  if (nextOnly) {
+    return sessionDates[0] || null;
```
Comment (harveysanders) [inline src/hooks/useInfoSession.ts:17]:
```suggestion
    return sessionDates[0] ?? null;
```

**Review by harveysanders — APPROVED** [review]

File: pages/programs/workforce.tsx (hunk 1 above)
Comment (ptbarnum4) [reply pages/programs/workforce.tsx:5]: I agree. I will make an issue to refactor momentjs out where possible. I use moment-timezone for the alert bar to ensure timezone is set to CST but the rest can go

File: pages/programs/workforce.tsx (hunk 2 above)
Comment (ptbarnum4) [reply pages/programs/workforce.tsx:126]: good catch! I had it 90% done, just forgot to implement

File: src/hooks/useInfoSession.ts (hunk 3 above)
Comment (ptbarnum4) [reply src/hooks/useInfoSession.ts:17]: ClipWindowsGIF

**Pull request merged**